addons/account/models/account_move.py

```diff
@@ -158,6 +158,10 @@ def _search_default_journal(self, journal_types):
     statement_line_id = fields.Many2one(
         comodel_name='account.bank.statement.line',
         string="Statement Line", copy=False, check_company=True)
+    use_storno_accounting = fields.Boolean(
+        string="Use Storno Accounting",
+        store=True, readonly=False,
```

Comment:
why do you need it stored on the account.move? there's no way a move could be using storno and another not... is it?

Reply:
I was confused too, but what he means with this is "this move has negative debit/credits because it is a debit/credit note", not that the accounting is in Storno.

Its a bit confusing, that's why I proposed to change the name of this variable; if needed at all.

Reply:
@qdp-odoo This field is more `use_negative_debit_credit`